lib/hex_web/email/ses.ex

```diff
@@ -3,8 +3,21 @@ defmodule HexWeb.Email.SES do
   require Logger
 
   def send(to, subject, body) do
-    HexWeb.Throttle.wait(HexWeb.SESThrottle)
-    do_send(to, subject, body)
+    ses_rate = Application.get_env(:hex_web, :ses_rate) |> String.to_integer
+
+    recipient_chunks(to, ses_rate)
+    |> Enum.each(fn recipients ->
```

Comment:
Do not pipe into a multi-line function call. Instead do: `Enum.each(recipient_chunks(to, ses_rate), fn recipients ->`.